### PR TITLE
Parameter typo fix for New-AZAutomationUpdateManagementAzureQuery

### PR DESCRIPTION
**Pull request by jemex**
## Checklist

- [x ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ x] The title of the PR is clear and informative
- [ x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Automation/Automation/Cmdlet/UpdateManagement/NewAutomationUpdateManagementAzureQuery .cs [hunk 1]
```diff
@@ -37,6 +38,10 @@ public class NewAutomationUpdateManagementAzureQuery : AzureAutomationBaseCmdlet
         [Parameter(Mandatory = false, HelpMessage = "List of locations for azure virtual machines.")]
         public string[] Locaton { get; set; }
 
+        [Parameter(Mandatory = false, HelpMessage = "List of locations for azure virtual machines.")]
+        [LocationCompleter("Microsoft.Automation/SoftwareUpdateConfiguration")]
+        public string[] Location { get; set; }
```
Comment (MiYanni) [inline src/Automation/Automation/Cmdlet/UpdateManagement/NewAutomationUpdateManagementAzureQuery .cs]: Simply add an `[Alias("Locaton")]` to `Location` and remove the `Locaton` property.

Comment (jemex) [reply src/Automation/Automation/Cmdlet/UpdateManagement/NewAutomationUpdateManagementAzureQuery .cs]: Thank you @MiYanni , this is more simpler

File: src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.cs [hunk 2]
```diff
@@ -117,5 +117,14 @@ public void TestGetSoftwareUpdateConfigurationRunWithPrePost()
             TestRunner.RunTestScript("Test-GetSoftwareUpdateConfigurationRunWithPrePost");
         }
 
+        [Fact]
+        [Trait(Category.Service, Category.Automation)]
+        [Trait(Category.RunType, Category.LiveOnly)]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        [Trait(Category.AcceptanceType, Category.BVT)]
+        public void TestGetSoftwareUpdateConfigurationAzureDynamicGroupLocationParameterBackwardCompatiple()
```
Comment (MiYanni) [inline src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.cs]: nit: Typo for `Compatible`

File: src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.cs [hunk 3]
```diff
@@ -117,5 +117,14 @@ public void TestGetSoftwareUpdateConfigurationRunWithPrePost()
...
+        {
+            TestRunner.RunTestScript("Test-CreateAndGetSoftwareUpdateConfigurationWithAzureDynamicGroupsWithLocationParameterbackwardCompatiple");
```
Comment (MiYanni) [inline src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.cs]: nit: Typo for `Compatible`

File: src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1 [hunk 4]
```diff
@@ -295,6 +295,62 @@ Test-CreateAndGetSoftwareUpdateConfigurationWithAzureDynamicGroupsOnlyWithOutTag
...
+    Assert-NotNull $sucGet.UpdateConfiguration.Targets.AzureQueries "Update targets  azureQueries list  null"	
+    Assert-AreEqual $sucGet.UpdateConfiguration.Targets.AzureQueries.Count  1 "Update targets  doesn't have the correct number of azure queries"
+   
+ }
+
+ <#
+.SYNOPSIS
+Test-CreateAndGetSoftwareUpdateConfigurationWithAzureDynamicGroupsOnlyWithOutTags
+#>
+ function Test-CreateAndGetSoftwareUpdateConfigurationWithAzureDynamicGroupsWithLocationParameterbackwardCompatiple
```
Comment (MiYanni) [inline src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1]: nit: Typo for `Compatible`

**Pull request merged**